otherlibs/systhreads/st_dummy.c

```diff
@@ -0,0 +1,12 @@
+/* 'otherlibs/threads' used to contain C support code for the OCaml
+   Thread module, which has since been migrated into the OCaml runtime itself.
+
+   To avoid breaking build scripts all over the place, we want to keep
+   building a C library archive, even if we don't have C code to
+   compile anymore. Windows and OSX don't really support creating
```

Comment:
```suggestion
/* 'otherlibs/systhreads' used to contain C support code for the OCaml
   Thread module, which has since been migrated into the OCaml runtime itself.

   To avoid breaking build scripts all over the place, we want to keep
   building a C library archive, even if we don't have C code to
   compile anymore. Windows and macOS don't really support creating
```